app/modules/update-tools.js

```diff
@@ -10,7 +10,7 @@ var updateTools = {
   updatesForDisplay: function(quantity) {
     var self = this;
 
-    if(quantity > quantities.updates) {
+    if(!quantity || quantity > quantities.updates) {
```

Comment:
minor stylistic niggle - had to read this three times before I was sure what it was doing. Maybe add some brackets around the (quantity > quantities.updates) to be explicit, rather than relying on operator precedence?